components/core/src/clp_s/ArchiveWriter.cpp

```diff
@@ -107,8 +107,17 @@ void ArchiveWriter::close() {
         header_and_metadata_writer.close();
     }
 
+    ArchiveStats archive_stats{
+            m_id,
+            m_timestamp_dict.get_begin_timestamp(),
+            m_timestamp_dict.get_end_timestamp(),
+            m_uncompressed_size,
+            m_compressed_size,
+            archive_range_index,
+            is_split
+    };
     if (m_print_archive_stats) {
-        print_archive_stats();
+        std::cout << archive_stats.as_string() << std::endl;
```

Comment:
```suggestion
        std::cout << archive_stats.as_string() << '\n';
```

To resolve clang-tidy violation.